pelita/game.py

```diff
@@ -81,18 +81,101 @@ def controller_exit(state, await_action='play_step'):
+              a list specifying the two teams to play the match with. The list
+              items team0 and team1 must be either
+               - a function with signature move(bot, state) -> (x, y), state
+                 the function takes a bot object and a state and returns the next
+                 position (x, y) of bot and state.
+                 state can be an arbitrary Python object, None by default
+               - the path to a Python module that defines at least a function
+                 called 'move' (specified as above), and a string TEAM_NAME.
+
+              When the team_specs are functions a "local" game will be run, when
+              they are paths to Python modules a "remote" game will be run instead.
+              See below in section Notes for more details about local and remote games.
+
+    layout_dict : dict
+               a dictionary representing a maze, as returned by pelita.layout.parse_layout
+    layout_name : str
+               a name for the layout (will be used in the UI).
+
+    max_rounds : int
+              The maximum number of rounds to play before the game is over. Default: 300.
+
+    seed : int
+        seed used to initialize the random number generator.
+
+    max_team_errors : int
+                   The maximum number of non fatal errors for a team before the
+                   game is over and the team is disqualified. Non fatal errors are
+                   timeouts and returning an illegal move. Fatal errors are raising
+                   Exceptions. Default: 5.
+
+    timeout_length : int or float
+                  Time in seconds to wait for the move function (or for the remote
+                  client) to return. After timeout_length seconds are elapsed an
+                  non-fatal error is recorded for the team.
+
+    viewers : list[viewer1, viewer2]
+           List of viewers to attach to the game. Implemented viewers: 'ascii',
+           'progress', tk'. If None, no viewer is attached.
+
+    viewer_options : do not use!
+
+    store_output : False or str
+               if store_output is a string it will be interpreted as a path to a
+               directory where to store stdout and stderr for the client processes.
+               It helps in debugging issues with the clients.
+
+    team_names : tuple(team_name_0, team_name_1)
+              a tuple containing the team names. If not given, names will be taken
+              from the team module TEAM_NAME variable or from the function name.
+
+    allow_exceptions : bool
+                    when True, allow teams to raise Exceptions. This is especially
+                    useful when running local games, where you typically want to
+                    see Exceptions do debug them. When running remote games,
+                    allow_exceptions should be False, so that the game master can
+                    collect the exceptions and cleanly create a game-over state if
+                    needed.
+
```

Comment:
Indendation is a little inconsistent. Either one or two characters left of the : 
Maybe ignore the : and just make it 4 everywhere?